ibm_catalog.json

```diff
@@ -275,7 +275,7 @@
                             {
                                 "diagram": {
                                     "caption": "Secure infrastructure on VPC for regulated industries",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/patterns/diagrams/svgs/vsi/vsi-vsi.drawio.svg",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/terraform-ibm-landing-zone/patterns/diagrams/svgs/vsi/vsi-vsi.drawio.svg",
```

Comment:
This is not correct - Ill fix in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/297